src/test.js

```diff
@@ -491,6 +493,13 @@ function synchronize( callback, priority ) {
 
 	if ( priority ) {
 		config.queue.splice( priorityCount++, 0, callback );
+	} else if ( random ) {
+		if ( !randomNumber ) {
+			randomNumber = randomNumberGenerator( random );
```

Comment:
"Unit" is intended to be understood in the [mathematical sense](https://en.wikipedia.org/wiki/Unit_interval), but I'm not married to it. However, an ideal name should capture both the "seeded" detail and indicate uniform distribution over the [0, 1) output range.